src/test/java/com/skypro/simplebanking/controller/IntegrationTests.java

```diff
@@ -0,0 +1,126 @@
+package com.skypro.simplebanking.controller;
+
+import com.skypro.simplebanking.dto.CreateUserRequest;
+import com.skypro.simplebanking.dto.UserDTO;
+import com.skypro.simplebanking.entity.User;
+import com.skypro.simplebanking.repository.AccountRepository;
+import com.skypro.simplebanking.repository.UserRepository;
+import org.json.JSONObject;
+import org.junit.Test;
```

Comment:
Нужно использовать 5й junit
```suggestion
import org.junit.jupiter.api.Test;
```